setup.py

```diff
@@ -52,8 +53,18 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-PACKAGE_DATA = {'pydicom': ['tests/test_files/*', 'tests/charset_files/*']}
+PACKAGE_DATA = {'pydicom': ['tests/test_files/*',
+                            'tests/charset_files/*']}
 REQUIRES = []
```

Comment:
Are the tests/test_files/ here still needed? They are already in the data files now.